docs/sources/developers/plugins/migration-guide/v6.x-v7.x/_index.md

```diff
@@ -6,18 +6,20 @@ keywords:
   - migration
   - plugin
   - documentation
-title: Migrating plugins from Grafana version 6.x to 7.x
+title: Migrate plugins from Grafana version 6.x to 7.x
 menutitle: v6.x to v7.x
 weight: 2500
 ---
 
-# Migrating plugins from Grafana version 6.x to 7.0
+# Migrate plugins from Grafana version 6.x to 7.0
+
+Follow the instructions in this section to upgrade Grafana from version 6.x to 7.0.
 
 ## What's new in Grafana 7.0?
 
-Grafana 7.0 introduced a whole new plugin platform based on React. The new platform supersedes the previous Angular-based plugin platform.
+Grafana 7.0 introduced a whole new plugin platform based on React. This new platform supersedes the previous Angular-based plugin platform.
 
-Plugins built using Angular still work for the foreseeable future, but we encourage new plugin authors to develop with the new platform.
+Plugins built using Angular still work in the near term, but we strongly encourage new plugin authors to develop with the new platform.
```

Comment:
Given below you've added a note on toolkits deprecation, we should add similar for Angular here, suggestion:

"Support for Angular was deprecated in Grafana 9. No new plugin submissions based on Angular will be accepted in the Plugin Catalog"

Reply:
fixed